.github/workflows/phpunit.yml

```
@@ -20,23 +20,20 @@ jobs:
     strategy:
       matrix:
         drupal-version:
-          - '10.1'
+          - '10.2'
         php-version:
           - '8.2'
         experimental: [false]
         do-leniency: [true]
-        patches: ["{}"]
-        include:
-          - drupal-version: '10.2'
-            php-version: '8.2'
-            experimental: true
-            do-leniency: true
-            patches: |-
+        patches: [
+          |-
               {
                 "drupal/core": {
                   "Referencable unpublished nodes": "https://www.drupal.org/files/issues/2024-01-07/2845144-78.patch"
                 }
               }
+        ]
```

Took for a spin in https://github.com/discoverygarden/embargo/pull/42

Apparently, it's failing to parse here. Could possibly change up as:

```suggestion
        patches:
          - |-
              {
                "drupal/core": {
                  "Referencable unpublished nodes": "https://www.drupal.org/files/issues/2024-01-07/2845144-78.patch"
                }
              }
```